utils/Api.js

```diff
@@ -0,0 +1,110 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    return Promise.all([
+      this.getInitialCards(),
+      this.getUserInfo(),
+      this.editUserInfo,
+    ]);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    });
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
```

Comment:
`    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
`


Please, reuse `${this._baseUrl}` everewhere, not to hardcode the base url